src/content/how-to/recorrer-array-javascript.md

```diff
@@ -0,0 +1,202 @@
+# Recorrer Arrays Javascript
+
+## Arrays y for loops
+
+Los _Arrays_ o arrays en Javascript ayudan a agrupar múltiples elementos de manera secuencial y proporcionan maneras de acceder a la información en base a su posición, iniciando desde el 0. 
+Los _for loops_ son una herramienta particularmente útil para realizar acciones sobre valores almacenados en un array. Los _for loops_ permiten programar bloques de código que se ejecutan de manera iterativa.
+Un ejemplo sencillo para visualizar estos dos conceptos en conjunto, podemos imaginarnos un array llamada `years` que contiene años donde ha ocurrido una operación.
+
+```js
+const years = [2018, 2019, 2020, 2021, 2022];
+
+for (let i = 0; i < years.length; i++) {
+    console.log(years[i]);
+}
+
+// Output:
+// 2018
+// 2019
+// 2020
+// 2021
+// 2022
+```
+Este código imprime en la consola todos los valores guardados adentro de `years`.
+
+## Funciones para recorrer Arrays
+
+Javascript provee muchas herramientas para recorrer arrays, optimizando acciones sobre los mismos, a continuación, se mencionan los mas comunes y versátiles.
+
+### For Loop
+
+Es el mas sencillo y genérico. Como se menciona anteriormente, permite ejecutar un bloque de código un numero determinado de veces. La sintaxis básica de un _for loop_ es el siguiente:
+
+```js
+for (inicialización; condición; iteración) {
+    código a ejecutar; 
+}
+```
+
+El for loop tiene 4 elementos principales:
+
+1. Inicialización: se especifica la variable que controla la operación del for.
+2. Condición: la condición controla cuando se deja de ejecutar el for.
+3. Iteración: determina la modificación sobre la variable control para controlar el flujo.
+4. Código: se programa el código que se va a ejecutar cada vez que la condición se siga cumpliendo.
+
+En este ejemplo se pondrá todo en práctica:
+
+Se tienen unos números guardados dentro de un array `numeros` y se quiere sumar 1 a todos los elementos dentro del array y luego imprimirlos a la consola 
+
+```js 
+let numeros = [5,8,13,6,9]
+
+for (let index = 0; index < numeros.length; index++) {
+    console.log(numeros[index]+1)
+}
+
+//output
+//6
+//9
+//14
+//7
+//10
+```
+
+### For In
+
+For… In se utiliza principalmente para recorrer todas las propiedades de un objeto. 
+
+```js
+const vehiculo = {
```

Comment:
Cambiar ejemplo para utilizar for in pero con arrays, recuerda que el articulo es sobre arrays y no objetos